test/factories/supervisor_reports.rb

```diff
@@ -2,7 +2,6 @@
 
 FactoryGirl.define do
   factory :supervisor_report do
-    association :user, factory: %i[user supervisor]
```

Comment:
I wish I could take the credit, but really it was just that my seed file didn't run. This fixed that.